src/syntax_highlighting.nim

```diff
@@ -327,17 +331,16 @@ proc parseNimCode*(s: Text, state: NimParseState, len = 100): tuple[segments: se
     elif r == rune"'":             chr
     elif r == rune"#":             comment
     elif r in "0123456789".runes:  number
-    
+    elif r in opRunes:
+      add sOperatorWord, 1
```

Comment:
shouldn't it be `sOperator` instead?

Reply:
fixed, didn't aware there's `sOperator` kind.